extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -70,11 +71,13 @@ export default compose( [
 
 		return {
 			planName: get( plan, [ 'product_name_short' ] ),
+			planPathSlug,
 			upgradeUrl,
 		};
 	} ),
-	withDispatch( ( dispatch, { upgradeUrl } ) => ( {
+	withDispatch( ( dispatch, { planPathSlug, upgradeUrl } ) => ( {
```

Comment:
(use plan slug for tracks, see my other comment)
```suggestion
	withDispatch( ( dispatch, { plan, upgradeUrl } ) => ( {
```